content/docs/get-started/azure/review-project.md

```diff
@@ -110,16 +110,31 @@ This Pulumi program creates an Azure resource group and storage account and expo
 **Note**: In this program, the location of the resource group is set in the configuration setting `azure:location` (check the `Pulumi.dev.yaml` file). This is an easy way to set a global location for your program so you don't have to specify the location for each resource manually. The location for the storage account is automatically derived from the location of the resource group. To override the location for a resource, simply set the location property to one of Azure's [supported locations](https://azure.microsoft.com/en-us/global-infrastructure/locations/).
 
 {{% lang python %}}
+
+> *Note*: As a prerequisite, [install virtualenv](https://virtualenv.pypa.io/en/latest/installation/) to manage project requirements
+
 For Python, before we deploy the stack, the following commands need to be run to create a virtual environment, activate it, and install dependencies:
 
+Create a virtual environment:
+
 ```bash
-$ virtualenv -p python3 venv
+$ python -m venv azure-env
 ```
 
+Activate the environment in Linux and MacOS:
+
 ```bash
-$ source venv/bin/activate
+$ source azure-env/bin/activate
```

Comment:
Do we really want to use names like `azure-env` here vs a more generic `venv`?  I don’t know the namespacing of these names - but I can’t come up with a reason why `azure-env` feels right for this (either it’s too specific or not specific enough).

Reply:
We should use `venv` for the reasons I commented earlier in https://github.com/pulumi/docs/pull/2184#discussion_r360544638

Reply:
venv is very generic and also the name of the executable. Because python can be sensitive to dependencies, it's generally good practice to create a virtual environment for each project. Here's the example from venv

`c:\>python -m venv c:\path\to\myenv`

We can go with just `myenv` to reduce confusion,

Reply:
Whatever we choose, we'll need to update the `.gitignore` file for all of our Python templates to exclude the virtual environment directory.

Is `venv` really so bad? It seems like it's commonly used based on GitHub's default `.gitignore` for Python:

https://github.com/justinvp/test123/blob/ab56476cad18c989f67d573033d80f16967fd2a5/.gitignore#L104-L111

```
# Environments
.env
.venv
env/
venv/
ENV/
env.bak/
venv.bak/
```

Reply:
OK, not my hill. `python3 -m venv venv` it is